src/system/Icons.cpp

```diff
@@ -7,6 +7,8 @@
 #include <filesystem>
 #include <fstream>
 
+#include <glib.h>
```

Comment:
I will genuinely ban you from all hypr repos if you include glib in my code again